build.sbt

```diff
@@ -10,16 +10,29 @@ resolvers ++= Seq(
 
 libraryDependencies ++= Seq(
   "ch.qos.logback"           % "logback-classic"          % "1.1.2",
+  "com.rojoma"              %% "rojoma-json-v3"           % "3.3.0",
+  "com.rojoma"              %% "rojoma-json-v3-jackson"   % "1.0.0"
+    exclude("com.rojoma", "rojoma-json-v3_2.10"),
   "com.rojoma"              %% "simple-arm-v2"            % "2.1.0",
-  "com.socrata"             %% "socrata-http-client"      % "3.3.0",
-  "com.socrata"             %% "socrata-http-jetty"       % "3.3.0",
+  "com.socrata"             %% "socrata-http-client"      % "3.3.0"
+    exclude("com.rojoma", "rojoma-json_2.10")
```

Comment:
The excludes are handled by ivy, and it works based on the Java name of the package.